infra/metadata_harmonizer_step_functions/lambdas/harmonize_metadata.py

```
@@ -0,0 +1,873 @@
+    - Traces are very large (long-running Lambdas with many agent interactions)
+    - Lambda shutdown interrupts the multipart upload
+    - Network issues during upload
+
+    These errors are logged at DEBUG level since they don't affect Lambda execution.
+    """
+    if HAS_LANGSMITH and get_langsmith_client:
+        try:
+            client = get_langsmith_client()
+            client.flush()
+            logger.info("LangSmith traces flushed successfully")
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)